src/network/message.rs

```diff
@@ -70,7 +70,7 @@ impl Encodable for CommandString {
         let mut rawbytes = [0u8; 12];
         let strbytes = self.0.as_bytes();
         if strbytes.len() > 12 {
-            return Err(encode::Error::UnrecognizedNetworkCommand(self.0.clone().into_owned()));
+            return Err(encode::Error::UnrecognizedNetworkCommand(self.0.clone().into_owned(), strbytes.len()));
```

Comment:
I think we should give this a separate error variant.

Reply:
How about `NetworkCommandTooLong`? or just a more generic `InvalidNetworkCommand`?

Reply:
I think `TooLong` is the better one. That's been the only invalidity condition for many years and I can't imagine it changing :).